cmake/SSGCommon.cmake

```diff
@@ -238,16 +238,29 @@ macro(_ssg_build_remediations_for_language PRODUCT LANGUAGES)
     foreach(LANGUAGE ${LANGUAGES})
       file(GLOB EXTRA_LANGUAGE_DEPENDS "${CMAKE_CURRENT_SOURCE_DIR}/fixes/${LANGUAGE}/*")
       file(GLOB EXTRA_SHARED_LANGUAGE_DEPENDS "${SSG_SHARED}/fixes/${LANGUAGE}/*")
+      set(ALL_FIXES_DIR "${CMAKE_CURRENT_BINARY_DIR}/all_fixes/${LANGUAGE}")
```

Comment:
Can we not just put all remediation scripts in the `fixes` directory rather than just having `all_fixes` and `fixes`?

Reply:
Yes, I agree, I also don't like it.